packages/api/src/command/medical/patient/update-patient.ts

```diff
@@ -102,5 +104,5 @@ export async function updatePatientWithoutHIEs(
 
   if (emit) patientEvents().emitUpdated(result);
 
-  return result;
+  return result.dataValues;
```

Comment:
:smart:

This makes me want to add an item to our code standards: 

> If the Command returns a Sequelize Model, but the return type of the function is the domain/interface and _not_ the Model, you MUST return the `.dataValues` of the Model.

For reference: why? because we can't deconstruct a Model, and if the return type is not a Model, this is not obvious and can lead to bugs as we modify the code.